garage/contrib/ros/robots/sawyer.py

```diff
@@ -140,9 +151,12 @@ def get_observation(self):
             list(self._limb.joint_efforts().values()))
 
         obs = np.concatenate(
-            (gripper_pos, gripper_ori, gripper_lvel, gripper_avel,
-             gripper_force, gripper_torque, robot_joint_angles,
+            (endpoint_pos, endpoint_ori, endpoint_lvel, endpoint_avel,
```

Comment:
To match the mujoco environment we have, for torque/effort control we used joint_angles, joint_vel and end effector position as observation space.

Reply:
Yes. But at least for environments like reacher we don't need that many fields as observation(e.g. endpoint_ori). I think a good way to solve this is have a get_full_observation for the sawyer robot class and customizes the environment observations.

Reply:
This interface doesn't have any use if it doesn't match the simulation environment. Please make it match. If the mujoco environment needs to have things/options added, please do that. They are a matching pair.